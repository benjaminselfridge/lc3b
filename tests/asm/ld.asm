        0x3010
        LEA     R0      DATA1
        LDW     R1      R0      0x0
        LDB     R2      R0      0x0
        LDB     R3      R0      0x1
        LDW     R4      R0      0x0
        LDW     R4      R0      0x1
        TRAP 0x0


DATA1:  0x1234
