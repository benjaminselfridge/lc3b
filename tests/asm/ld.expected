[ RegContains 1 0x1234
, RegContains 2 0x34
, RegContains 3 0x12
, RegContains 4 0x0000
]